synapse/models/infotech.py

```diff
@@ -381,39 +381,39 @@ def getModelDefs(self):
                     'doc': "A vulnerability scan result for an asset."}),
 
                 ('it:mitre:attack:status', ('str', {'enums': 'current,deprecated,withdrawn'}), {
-                    'doc': 'A Mitre ATT&CK element status.',
+                    'doc': 'A MITRE ATT&CK element status.',
```

Comment:
I thought the preferred form was `MITRE` ?

Reply:
That's the change (Mitre->MITRE)...?

Reply:
nothing to see here..